ractor.rb

```diff
@@ -83,7 +266,51 @@ def self.select(*ractors, yield_value: yield_unspecified = true, move: false)
     }
   end
 
-  # Receive an incoming message from Ractor's incoming queue.
+  # Receive an incoming message from the current Ractor's incoming port's queue, which was
+  # sent there by #send.
+  #
```

Comment:
It wasn't clear to me that `Ractor.receive` and `Ractor#receive` have the same effect. Should we mention it?